pure_pagination/paginator.py

```diff
@@ -168,7 +170,7 @@ def end_index(self):
     @add_page_querystring
     def pages(self):
         if self.paginator.num_pages <= PAGE_RANGE_DISPLAYED:
-            return range(1, self.paginator.num_pages+1)
+            return list(range(1, self.paginator.num_pages+1))
         result = []
         left_side = PAGE_RANGE_DISPLAYED/2

```

Comment:
I think that for full compatibility with Python 3, you must add an `//2` instead of `/2` in this lines.